crates/holochain_zome_types/src/zome_io.rs

```diff
@@ -308,10 +308,15 @@ pub struct ZomeCallUnsigned {
     pub expires_at: Timestamp,
 }
 
-impl ZomeCallUnsigned {
-    /// Prepare the canonical bytes for an unsigned zome call so that it is
+// type SerializedBytesAndHash = (Arc<[u8])
+
+impl ZomeCallParams {
+    /// Prepare the canonical bytes for zome call parameters so that they are
     /// always signed and verified in the same way.
```

Comment:
This is no longer quite so true right? Any valid msgpack serialisation can be used for a call, there's no longer a "one way and one way only" situation

Reply:
Right, they're not canonical in that sense that only Holochain serialization can produce them.

Reply:
Should the doc comment be updated here?